src/neuroglancer/segmentation_user_layer.ts

```diff
@@ -157,6 +161,21 @@ export class SegmentationUserLayer extends Base {
       this.specificationChanged.dispatch();
     });
 
+    verifyObjectProperty(specification, SEGMENT_STATED_COLORS_JSON_KEY, y => {
+      if (y !== undefined) {
+        let {segmentEquivalences} = this.displayState;
+        // Parse the color as a 32-bit integer here, because the Uint64.parseString()
+        // function does not seem to handle strings of the form '0xRRGGBB'.
+        // And support a CSS color "#RRGGBB" by turning into "0xRRGGBB".
+        let result = verifyObjectAsMap(y, x => parseInt(String(x).replace(/^#/, '0x'), 0));
```

Comment:
You can use parseRGBColorSpecification or parseRGBAColorSpecification in util/color.ts to handle arbitrary CSS color specifications.  Handling alpha might be a bit more work but would probably be quite useful -- but I think that would be reasonable to put off if you don't want to do it now.

Reply:
Thanks for the suggestion.  I just pushed an update using parseRGBColorSpecification.  I think I will put off looking into alpha handling.